assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -33,13 +33,13 @@ che.websocket.internal.endpoint=NULL
 # workspace. This is the directory in the machine where your projects are placed.
 che.workspace.projects.storage=/projects
 
-# Used when {orch-name}-type components in a devfile request project PVC creation
-# (Applied in case of `unique` and `per workspace` PVC strategy. In case of the `common` PVC strategy,
-# it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.)
+# Used when {orch-name}-type components in a devfile request project Persistent Volume Claim creation.
+# Applied in case of `unique` and `per workspace` PVC strategy. In case of the `common` PVC strategy,
+# it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.
 che.workspace.projects.storage.default.size=1Gi
 
 # Defines the directory inside the machine where all the workspace logs are placed.
-# Provide this value into the machine, for example, as an environment variable.
+# Provides this value into the machine, for example, as an environment variable.
```

Comment:
This should likely stay `Provide` as it's an instruction to the user on how to use the value.